### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by temikfart**
Added:
- Table of Contents;
- readme draft (some paragraphs marked as TODO);
- WebRTC Protocol info;
- Grabber description (How to build, run, configure);
- Signaling description (How to build, run, configure);
- Turn description (How to build, run);
- MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,170 @@
...
+connect peers from the “blue” network with computers from the “red” network.
 
-```
-npm ci
-npm run start:dev
-```
+The solution consisted of 3 main parts:
 
-or as docker image
+- **Grabber**
```
Comment (kbats183) [inline README.md:64]: Let describe parts of sevice:

* Grabber - standalone application for background capturing and sharing sceen and webcam
* Signalling - webserver to exchange connection data between the grabber and players, capture video in browser and maintain a list of active grabbers
* Turn - server for forwarding video packages between grabber and players when thay separated usin nat

File: README.md [hunk 2]
```diff
@@ -1,92 +1,170 @@
...
-## Signalling
----
+WebRTC works by connecting each peer with each other to achieve peer to
+peer communication.
+In our example, peer A would be the team computer, and peer B would be our
+computer for live-streaming (since it’s the one that’s actually showing the
+feed).
 
-A socket.io node js server for signaling between the players and grabbers.
+<img src="img/webrtc-example.png">
```
Comment (kbats183) [inline README.md:44]: May be this picture isn't necessary

**Review by kbats183 — APPROVED** [review]
Greate

**Review by kbats183 — APPROVED** [review]
Greate

File: README.md (hunk 2 above)
Comment (temikfart) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (temikfart) [reply README.md]: Added

**Pull request merged**